src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,23 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static final SessionFactory INSTANCE = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new RuntimeException("Can`t create session factory");
```

Comment:
In the `catch` block of the `initSessionFactory` method, consider including the original exception `e` as a cause in the `RuntimeException`. This can be done by passing `e` as the second argument to the `RuntimeException` constructor, which will provide more context for debugging purposes.